src/org/labkey/test/pages/core/admin/ShowAdminPage.java

```diff
@@ -106,6 +106,12 @@ public void clickEmailCustomization()
         clickAndWait(elementCache().emailCustomizationLink);
     }
 
+    public void clickExperimentalFeatures()
+    {
+        goToSettingsSection();
+        clickAndWait(elementCache().experimentalfeaturesLink);
+    }
+
     public void clickNotificationServiceAdmin()
```

Comment:
Should we be returning an ExperimentalFeatures page object that has methods to enable and disable experimental features by name? Seems like that would be a generally useful feature for others needing test automation.

Reply:
Created the page and checked in